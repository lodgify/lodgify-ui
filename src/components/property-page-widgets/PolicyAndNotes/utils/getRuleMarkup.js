import React from 'react';

import { buildKeyFromStrings } from 'utils/build-key-from-strings';
import { Paragraph } from 'typography/Paragraph';

/**
 * @param  {string} rule
 * @param  {number} index
 * @return {Object}
 */
export const getRuleMarkup = (rule, index) => (
  <Paragraph key={buildKeyFromStrings(rule, index)} size="medium">
    {rule}
  </Paragraph>
);
