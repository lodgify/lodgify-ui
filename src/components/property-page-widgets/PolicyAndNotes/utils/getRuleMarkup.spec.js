import { getRuleMarkup } from './getRuleMarkup';

describe('`getRuleMarkup`', () => {
  it('should return the right structure', () => {
    const actual = getRuleMarkup('yo', 1);

    expect(actual).toMatchSnapshot();
  });
});
