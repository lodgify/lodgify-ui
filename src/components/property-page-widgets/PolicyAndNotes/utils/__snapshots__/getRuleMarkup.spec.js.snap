// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`getRuleMarkup\` should return the right structure 1`] = `
<Paragraph
  size="medium"
  weight={null}
>
  yo
</Paragraph>
`;
